ProteoformSuiteGUI/DisplayUtility.cs

```diff
@@ -47,21 +47,20 @@ public static void tooltip_graph_display(ToolTip t, MouseEventArgs e, Chart c, P
             }
         }
 
-        public static void GraphRelationsChart(Chart ct, List<ProteoformRelation> relations)
+        public static void GraphRelationsChart(Chart ct, List<ProteoformRelation> relations, string series)
         {
+            ct.Series[series].XValueMember = "group_adjusted_deltaM";
+            ct.Series[series].YValueMembers = "running_sum";
```

Comment:
Running sum should be the same as the group count, based on a quick check of the calculations you added... Maybe I'm missing something.